src/PowerShell.Core.Instrumentation/PowerShell.Core.Instrumentation.man

```diff
@@ -2186,10 +2187,12 @@
               />
         </events>
         <channels>
-          <!--There are two channels defined for Windows PowerShell instrumentation
+          <!--There are four channels defined for Windows PowerShell instrumentation
```

Comment:
Sorry, I should have mentioned this before, you'll want to remove these comments around Admin as well.